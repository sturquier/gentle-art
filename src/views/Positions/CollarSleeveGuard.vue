<template>
  <h1>Collar sleeve guard</h1>
</template>
