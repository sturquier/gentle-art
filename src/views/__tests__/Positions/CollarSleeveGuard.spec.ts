import { describe, it, expect } from 'vitest'
import { mount } from '@vue/test-utils'

import CollarSleeveGuard from '../../Positions/CollarSleeveGuard/CollarSleeveGuard.vue'

describe('CollarSleeveGuard', () => {
  it('renders properly', () => {
    const wrapper = mount(CollarSleeveGuard)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
