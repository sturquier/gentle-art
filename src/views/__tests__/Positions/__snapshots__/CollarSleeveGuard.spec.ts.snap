// Vitest Snapshot v1

exports[`CollarSleeveGuard > renders properly 1`] = `"<h1>Collar sleeve guard</h1>"`;
